web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx

```diff
@@ -1,6 +1,64 @@
-function SendCoins() {
-  // This is a placeholder
-  return <h1>Send coins </h1>;
-}
+import { useState } from 'react';
+import usePost from '../../../../custom/api/usePost';
+import { SendTransactionObject } from '../../../../models/AccountModel';
+import { Button } from '@massalabs/react-ui-kit';
+import { useLocation } from 'react-router-dom';
+import WalletLayout from '../../../../layouts/WalletLayout/WalletLayout';
+export default function SendCoins() {
+  const { state } = useLocation();
+  const nickname = state.nickname;
+  const { mutate, isSuccess } = usePost<SendTransactionObject>(
+    'accounts',
+    nickname,
+    'transfer',
+  );
+  if (isSuccess) {
+    console.log('Transfer success');
+  }
+
+  const [amount, setAmount] = useState<string>('64');
+  const [fees, setFees] = useState<string>('42');
```

Comment:
64? 42? 🤔  Curiosity: what is that exactly!?

Reply:
this is just a dummy front that calls the transfer from the back, the real front end will be implemented by @pivilartisant

Reply:
placeholder numbers ! I did the form and @Pierregvx  did the backend so we could work with out being stuck

Reply:
What means placeholder here!? This would be a initial content for the field!? I don't understand...

BTW, you are not aware of the <form> and useState. Let's sync in a call because it's recurrent. Let's clarify that in a call!